app/imports/collections/applicationSections.js

```diff
@@ -11,6 +11,12 @@ class ApplicationSectionsCollection extends Mongo.Collection {
     // This call it's parent original method.
     return super.insert(doc,callback);
   }
+
+  update(selector, modifier, ...optionsAndCallback) {
```

Comment:
This is the class we are extending
https://github.com/meteor/meteor/blob/devel/packages/mongo/collection.js#L552